src/test/scala/org/broadinstitute/dsde/firecloud/dataaccess/MockThurloeDAO.scala

```diff
@@ -12,12 +12,20 @@ import scala.util.{Success, Try}
  *
  */
 class MockThurloeDAO extends ThurloeDAO {
+  var nextGetProfileResponse:Option[Profile] = None
+  val testProfile = Profile("Rich", "Hickey", "CTO", None, "Datomic", "Data Storage",
+    "NYC", "NY", "USA", "David Mohs", "Not for no profit")
+
+  def reset() = {
```

Comment:
Instead of a reset method, I'd rather see a new instance of `MockThurloeDAO` for each test. Using a reset method makes it too easy to add state to the mock and forget to add code to the reset method which could cause unintended cross-talk between tests.